ToDoList/components/form/form.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { StyleSheet, keyboardType } from 'react-native';
+import { View, Text, Button } from 'native-base';
+import GenerateForm from 'react-native-form-builder';
+
+const fields = [
+    {
+        type: 'text',
+        name: 'todo',
+        required: true,
+        label: 'Adding a to do',
+    }
+];
+
+
+export default class AddToDo extends Component {
+    constructor(props) {
+        super(props);
+        this.addNewToDo = this.addNewToDo.bind(this);
+    }
+
+    // state = {
+    //     addNew: props.addItem
+    // }
+
+    addNewToDo() {
+        const formValues = this.formGenerator.getValues();
+        console.log('FORM VALUES: ', formValues);
+        // console.log('props: ', props)
```

Comment:
Always check to remove console.logs it's the first thing a developer will point out. Also any comment lines that aren't necessary for future reference should be removed.